java/jdbc/src/main/java/io/vitess/jdbc/ConnectionProperties.java

```diff
@@ -93,6 +93,24 @@
         null);
 
     // Vitess-specific configs
+    private StringConnectionProperty keyspace = new StringConnectionProperty(
```

Comment:
you probably want some getters for this and the below properties.  I.E. `getKeyspace()`